### ISSUE TITLE
dotnet_sort_system_directives_first and dotnet_separate_import_directive_groups (IDE0055) don't generate warnings

### ISSUE THREAD

**Issue by perlotane** [dotnet/roslyn#78391]
**Version Used**: 
Visual Studio 2022 `17.13.6` (same behaviour with Visual Studio 2022 `17.14.0 Preview 3`)
.NET `8.0.408` (with \<AnalysisLevel\>latest-all\</AnalysisLevel\>) (same behaviour with .NET `9.0.300-preview.0.25177.5`)
No extensions installed or settings changed made manually in the Preview version.

**Steps to Reproduce**:
minimal repro repo - <sorry, repo now deleted>

This repo includes an .editorconfig with:
```
dotnet_diagnostic.IDE0055.severity = warning
dotnet_sort_system_directives_first = true
dotnet_separate_import_directive_groups = false
```

and this at the top of one of the .cs files (TestClass.cs):
```
using EditorConfigIssue.NewFolder;

using System;
using System.Linq.Expressions;

namespace EditorConfigIssue
```

**Expected Behavior**:
Warnings for IDE0055 are generated in Intellisense and build, because the custom using comes before the System usings, and because there's an empty line between them.

**Actual Behavior**:
No warning is generated in either Intellisense or build. There is also no quick action available to fix them. 

Code Cleanup (when set to have only 'Fix all warnings and errors set in EditorConfig') actually does reorganise them though.
